packages/@vue/cli-plugin-e2e-nightwatch/generator/template/tests/e2e/specs/test.js

```diff
@@ -1,14 +1,21 @@
 // For authoring Nightwatch tests, see
-// http://nightwatchjs.org/guide#usage
+// https://nightwatchjs.org/guide
 
 module.exports = {
   'default e2e tests': browser => {
     browser
-      .url(process.env.VUE_DEV_SERVER_URL)
-      .waitForElementVisible('#app', 5000)
+      .init()
+      .waitForElementVisible('#app')
       .assert.elementPresent('.hello')
-      .assert.containsText('h1', 'Welcome to Your Vue.js <%- hasTS ? '+ TypeScript ' : '' %>App')
+      .assert.containsText('h1', 'Welcome to Your Vue.js App')
```

Comment:
```suggestion
      .assert.containsText('h1', 'Welcome to Your Vue.js <%- hasTS ? '+ TypeScript ' : '' %>App')
```

This is required to pass the typescript plugin's e2e test.